scripts/irods/test/test_iticket.py

```diff
@@ -1122,6 +1122,25 @@ def test_ticket_mtime_is_updated_when_ticket_properties_are_modified_by_non_admi
         self.assertGreater(modify_ts, create_ts)
         self.assertGreater(modify_ts, previous_ts)
 
+    def test_exceeding_ticket_use_count_returns_correct_error_code__issue_7967(self):
+        ticket_string = "issue_7967_uses_count"
+        data_object = "issue_7967_uses_filename"
+        self.user.assert_icommand(['itouch', data_object])
+
+        self.user.assert_icommand(['iticket', 'create', 'read', data_object, ticket_string])
+
+        # Modify ticket usage amount
+        self.user.assert_icommand(['iticket', 'mod', ticket_string, 'uses', '3'])
+        self.user.assert_icommand(['iticket', 'ls', ticket_string], 'STDOUT', ['uses count: 0', 'uses limit: 3'])
+        self.user.assert_icommand(['iget', '-f', '-t', ticket_string, data_object])
+        self.user.assert_icommand(['iticket', 'ls', ticket_string], 'STDOUT', ['uses count: 1'])
+        self.user.assert_icommand(['iget', '-f', '-t', ticket_string, data_object])
+        self.user.assert_icommand(['iticket', 'ls', ticket_string], 'STDOUT', ['uses count: 2'])
+        self.user.assert_icommand(['iget', '-f', '-t', ticket_string, data_object])
+        self.user.assert_icommand(['iticket', 'ls', ticket_string], 'STDOUT', ['uses count: 3'])
+        self.user.assert_icommand(['iget', '-f', '-t', ticket_string, data_object], 'STDERR', ['-892000 CAT_TICKET_USES_EXCEEDED'])
+        self.user.assert_icommand(['iticket', 'ls', ticket_string], 'STDOUT', ['uses count: 3'])
```

Comment:
does this mean it no longer goes to 4?     good if true.

Reply:
Sure does!

Reply:
excellent.